torch/_torch_docs.py

```diff
@@ -1465,6 +1465,47 @@ def merge_dicts(*dicts):
     tensor([ 0.5000, -0.4702, -0.4599,  0.5000])
 """.format(**common_args))
 
+add_docstr(torch.complex,
+           r"""
+complex(real, imag, out=None) -> Tensor
+
+Constructs a complex-valued tensor from Cartesian coordinates.
```

Comment:
Constructs a complex tensor with its real part equal to :attr:`real` and its imaginary part equal to :attr:`imag`.